source/Tests/Playnite.Tests/Database/FiltersDatabaseTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using NUnit.Framework;
+using Playnite.Common;
+using Playnite.Database;
+using Playnite.SDK.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Playnite.Tests.Database
+{
+    [TestFixture]
+    public class FiltersDatabaseTests
```

Comment:
Missing test that checks that already existing items will be sorted alphabetically even if they are not on the sort list. This will be the case for everyone updating from 10.9 to 10.10.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3258/commits/4d46394eb4383bbb9e60f82ef1798476a6dd17c4